app/data/dummy-apply-2-application.js

```diff
@@ -0,0 +1,14 @@
+// Copies dummy application and tweaks some fields
+var dummyApply2Application = JSON.parse(JSON.stringify(require('./dummy-application')))
+
+// Custom Apply 2 fields
+dummyApply2Application.apply2 = true
+dummyApply2Application.previousApplications = ['12345']
+
+// No sections completed
+dummyApply2Application.completed = {}
```

Comment:
`completed` or `reviewed`. Same to ensure parity with Apply 1 application?

Reply:
I was going to use the same data structure but change the language in the UI, so we reuse logic.